src/Microsoft.Management.Configuration.Processor/DscModules/DscModuleV2.cs

```diff
@@ -95,9 +101,20 @@ public IReadOnlyList<DscResourceInfoInternal> GetAllDscResources(PowerShell pwsh
             var getResult = pwsh.AddCommand(this.InvokeDscResourceCmd)
                                 .AddParameters(PrepareInvokeParameters(name, settings, moduleSpecification))
                                 .AddParameter(Parameters.Method, DscMethods.Get)
-                                .Invoke()
+                                .InvokeAndStopOnError()
                                 .FirstOrDefault();
 
+            if (pwsh.HadErrors)

```

Comment:
Helper method?  I don't understand why it was pulled out of the other one, but at least it could be one here to prevent the block from being copied three times.